frontend/src/AppRouter.tsx

```diff
@@ -243,17 +243,17 @@ const AppRouter: React.FC = () => {
           protected
           path="/testFileManagement"
           title={getTitle('Test')}
-          component={TestFileManagement}
+          customComponent={TestFileManagement}
           layout={AuthLayout}
         />
         <AppRoute
           exact
           path="/test/notes"
           title={getTitle('Test Notes')}
-          component={TestNotes}
+          customComponent={TestNotes}
```

Comment:
is there a clear reason why we should rename this from component -> customcomponent? I'm not a big fan of renames like this because I feel like it's just replacing dev A's personal preference with dev B's personal preference and we end up with bloated PRs.

Reply:
The reason is that the underlying Router that this is wrapping has a component parameter that was being hidden by this name. This was causing some confusion on type checking between the passed component and the component that Router has